CHANGELOG.md

```diff
@@ -77,6 +77,8 @@
   [#2642](https://github.com/Kong/kubernetes-ingress-controller/pull/2642)
 
 #### Fixed
+- Fixed the problem that logs from reporter does not appear in the pod log.
```

Comment:
You'll want to rebase that on `main` since #2638 was already merged and adds an entry about 2.4.2 here.

Reply:
done